kasparovd/controllers/transaction.go

```diff
@@ -101,9 +101,9 @@ func GetTransactionByHashHandler(txHash string) (interface{}, error) {
 // GetTransactionsByAddressHandler searches for all transactions
 // where the given address is either an input or an output.
 func GetTransactionsByAddressHandler(address string, skip uint64, limit uint64) (interface{}, error) {
-	if limit < 1 || limit > maxGetTransactionsLimit {
```

Comment:
Please reconsider this implicit behavior and if you insist on keeping it, please document it.